packages/cloud_firestore/cloud_firestore_web/lib/cloud_firestore_web.dart

```diff
@@ -54,11 +53,7 @@ class FirebaseFirestoreWeb extends FirebaseFirestorePlatform {
 
   @override
   Future<void> clearPersistence() async {
-    try {
-      await _webFirestore.clearPersistence();
-    } catch (e) {
-      throw getFirebaseException(e);
-    }
+    return guard(_webFirestore.clearPersistence);
```

Comment:
In those cases, you can remove the `async` keyword from the function

It's a slight performance boost.

Reply:
@russellwheatley This needs to be applied to all the modified functions, not just this one

Like disableNettwork/enableNetwork/...